File: CodeWars/8kyu/Get_Planet_Name_By_ID.py
# The function is not returning the correct values. Can you figure out why?
#
# Example (Input --> Output ):
#
# 3 --> "Earth"
# DEBUGGING
# Solution
def get_planet_name(id):
    switch = {
        1:"Mercury",
        2:"Venus",
        3:"Earth",
        4:"Mars",
        5:"Jupiter",
        6:"Saturn",
        7:"Uranus",
        8: "Neptune",
}
    return switch[id]

File: CodeWars/8kyu/Get_Planet_Name_By_ID.cpp
/*
The function is not returning the correct values. Can you figure out why?

Example (Input --> Output ):

3 --> "Earth"
Debugging
*/
// Solution
#include <string>

std::string get_planet_name(int id)
{
    std::string name;
    switch (id) {
    case 1: name = "Mercury"; break;
    case 2: name = "Venus"; break;
    case 3: name = "Earth"; break;
    case 4: name = "Mars"; break;
    case 5: name = "Jupiter"; break;
    case 6: name = "Saturn"; break;
    case 7: name = "Uranus"; break;
    case 8: name = "Neptune"; break;
    }
    return name;
}